src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -116,4 +123,23 @@ class Lambda {
     response.code == 204
   }
 
+  // Count the number of purge requests we are making
+  private def sendPurgeCountMetric: Unit = {
+    val metric = new MetricDatum()
+      .withMetricName("purges")
+      .withUnit(StandardUnit.None)
+      .withValue(1)
+
+    val putMetricDataRequest = new PutMetricDataRequest().
+      withNamespace("fastly-cache-purger").
+      withMetricData(metric)
+
+    try {
```

Comment:
Catch all exceptions as my confidence that this won't be blocked by AWS permissions on the first attempt is fairly low.